packages/valory/skills/abstract_round_abci/behaviour_utils.py

```diff
@@ -2234,6 +2233,7 @@ def _handle_unhealthy_tm(self) -> Generator:
                 # By setting "on_startup" to True in the reset_tendermint_with_wait() call above,
                 # wait_from_last_timestamp() will not be called at all.
                 yield from self.sleep(self.hard_reset_sleep)
+                self.gentle_reset_attempted = False
```

Comment:
Could you explain this change a bit?

Reply:
This is a flag we utilize so that the first time we try to recover we perform a "soft" reset, i.e., do not reset the node, and instead, we stop and start the node again.

This is now marked as not-attempted only after we successfully perform a hard reset.